src/v2/controllers/DeviceEnrollmentTerminalViewController.js

```diff
@@ -0,0 +1,42 @@
+import { _, Controller } from 'okta';
+import ViewFactory from '../view-builder/ViewFactory';
+import {FORMS as RemediationForms} from '../ion/RemediationConstants';
+
+export default Controller.extend({
+  className: 'deviceEnrollment-controller',
+
```

Comment:
set a flag that say this controller does not have contain any idx remediation responses and it is a terminal state.

`terminalStateWithNoRemediation : true`